46-sugar-ant/technical_specification.md

```diff
@@ -279,6 +288,8 @@ There are only the following two exceptions where the token will be also added b
   - the user must be already registered
   - the `id` field of the auth context will contain the internal id of the user, and it must correspond to the `user_id` in the path
 
+The `role` field is set in the "token exchange" code that converts the OIDC token to an internal access token. This code checks whether the current user is active and has a valid claim granting a GHGA role. In addition to checking the validity of the claim, the code now also needs to check whether the claim has the `iva_id` field set which is associated with an existing IVA in the verified state.
```

Comment:
Wasn't this already implemented before the latest change? Is this just a clarification of the epic doc?

Reply:
The first two sentences describe the situation before, the last one the change that needs to be implemented.

Until now role type claims did not have an associated IVA, only data access grant claims used it.

Reply:
Oh I see. Thanks for clarifying.